src/60_Samples_%26_Templates/Shopping_Cart.html

```diff
@@ -81,6 +96,7 @@ <h3>Your Basket is Empty. </h3>
     <form id="form-cart-delete" method="POST" target="_top" action-xhr="https://abe-cart-service.glitch.me/delete-cart-item" on="submit-success: AMP.setState({
                 cartItemsList: event.response
             })" novalidate="">
+        <input type="hidden" name="id" value="" [value]="cartItem.id">
         <input type="hidden" name="color" value="" [value]="cartItem.color">
```

Comment:
I think it'd be fine to only transmit the id, color and size shouldn't be needed for deletion. This would also mean that we can simplify the setState action to `AMP.setState({ itemToDelete: {{id}} })`.

Reply:
Thanks Sebastian, quick comment here: 
For this use case, "color" and "id" play the same role, but regarding "size": I'm using it as an identifier as well. For example: users can remove all "Medium" size items of id "1", but not the "Small" sizes of the same ID. So I think we can get rid of color, but size should stay. 
Thoughts?

Reply:
We could encode the size in the id. But maybe it's better then to leave it as it is...

Reply:
Just removed the "color" and updated the glitch service as well, to accept only product ID and size.